cmd/juju/commands/debuglog.go

```diff
@@ -189,22 +189,37 @@ func (c *debugLogCommand) Run(ctx *cmd.Context) (err error) {
 	if err != nil {
 		return err
 	}
+	writer := ansiterm.NewWriter(ctx.Stdout)
 	for {
 		msg, ok := <-messages
 		if !ok {
 			break
 		}
-		c.writeLogRecord(ctx.Stdout, msg)
+		c.writeLogRecord(writer, msg)
 	}
 
 	return nil
 }
 
-func (c *debugLogCommand) writeLogRecord(w io.Writer, r api.LogMessage) {
+var SeverityColor = map[string]*ansiterm.Context{
```

Comment:
Is this the same as what's in loggo? Are you duplicating to decouple from the default in loggo?